src/player.h

```diff
@@ -185,13 +191,23 @@ static const std::array< std::string, PMM_COUNT > player_movemode_str = { {
 class player : public Character
 {
     public:
+
         player();
         player( const player & ) = delete;
         player( player && );
         ~player() override;
         player &operator=( const player & ) = delete;
         player &operator=( player && );
 
+        /**
+         * Toggle attitude towards stolen goods, ask if UNSET
+         *
+         * UNSET: Ask player when attempting to steal
+         * HONEST: Do not pick up stolen items, even if selected in pickup
+         * STEAL: Pick up stolen items without query
+         */
+        thief_mode thief;
```

Comment:
Please use creature::values for musc state like this.
https://github.com/CleverRaven/Cataclysm-DDA/blob/9119b820d5503409c180c5e5bc71db4744ebb6c9/src/creature.h#L351

Reply:
Ah, thanks I will try that